### ISSUE TITLE
Pagination won't go over page 1000

### ISSUE THREAD

**Issue by BaptisteKaliop** [kuzzleio/kuzzle-backoffice#342]
The backoffice doesn't allow to go over 10K documents

![bo bug](https://user-images.githubusercontent.com/24288932/27821303-1c4d47d6-60a2-11e7-96be-b24cb3f891a0.png)

Looks like ES limit before scroll

**Comment by AnthonySendra** [comment]
Hello,

Yes this is a limitation from ES, without scroll you can't paginate all documents.
But, if we implement the scroll, you can only reach the next page from the current one.

We believe that nobody needs to see more than 1000 pages of results. The user will filter his search or order it before clicking 1000 times on a page ;)

**Comment by xbill82** [comment]
This may be worth a little text (box, pill, tooltip...) explaining what @AnthonySendra just wrote.

**Comment by samniisan** [comment]
I agree that the use case of going beyond 1,000 pages seems weird, the user should have filtered his search.

Although, we could also befenit from ES's `search_after` feature which allows to break this hard limit without using scrolls that "freeze" the result set.
https://www.elastic.co/guide/en/elasticsearch/reference/master/search-request-search-after.html

For this, we need to always have a sorting method, which is always the case (either user-specified or the default `_uid` sort).
